sdk/formrecognizer/azure-ai-formrecognizer/setup.py

```diff
@@ -44,6 +44,7 @@
     author='Microsoft Corporation',
     author_email='azpysdkhelp@microsoft.com',
     url='https://github.com/Azure/azure-sdk-for-python',
+    keywords="azure, form recognizer, cognitive services, document analyzer, document analysis",
```

Comment:
added `document analyzer` to kind of mimic the `form recognizer` search term not sure how relevant it will be for search terms.

Reply:
consider "applied ai" since it's not technically under CS anymore

Reply:
I like that, it's very relevant. Adding it!

Reply:
Just be clear - I didn't make that up. It's the name of the grouping under Azure: https://docs.microsoft.com/en-us/azure/applied-ai-services/